Automattic/pocket-casts-android
# Transcripts - Fallback to alternative transcript

## Description
This fallbacks to alternative format if main format fails while loading a transcript.

## Testing Instructions
1. Play an episode from chaosRadio (eg. https://pca.st/r2ct46up)
2. Open transcript view
3. ✅ Notice that an alternative format (`application/json`) is loaded as the main (`text/vtt`) one is empty 
4. Download another episode from chaosRadio
5. Wait for sometime
6. Disable internet
7. Play downloaded episode
8. Open transcript view for the downloaded episode
9. ✅ Notice that an alternative format (`application/json`) is loaded as the main (`text/vtt`) one is empty

## Screenshots or screencast

From player

https://github.com/user-attachments/assets/ac9c6765-01e3-4fe8-91c1-c55af11d800d

From download

https://github.com/user-attachments/assets/d2ad8f13-4fb8-46b4-a003-340422466617




## Checklist
 
#### I have tested any UI changes...
